docs/reference/content/guides/indexing.md

```diff
@@ -25,7 +25,10 @@ has two primary components to cover here:  `fields` and `options`.  An index def
 @Entity
 @Indexes({
     @Index(fields = @Field("field2", type = DESC)),
```

Comment:
This is actually incorrect as well.  It should be:

         @Index(fields = @Field(value = "field2", type = DESC)),

Would you mind fixing that as well?

Reply:
I added this to the branch